docs/understand_mmcv/io.md

```diff
@@ -92,7 +112,9 @@ d
 e
 ```
 
-Then use `list_from_file` to load the list from a.txt.
++ Load from disk
+
+Use `list_from_file` to load the list from a.txt.
 
 ```python
```

Comment:
Try whether we need indentation for code block to make it under the bullet of `Load from disk`.